### PR TITLE
Update tomcat nginx and tail plugins

### PR DESCRIPTION
**Pull request by ericwholt**
When using a parameter with relevant_if and required true. It can cause an issue when starting log agent, if no value is provided for the required parameter when relevant if is false. Stanza has been updated to allow default with required true which will allow this configuration
- Update `tomcat` plugin ([PR206](https://github.com/observIQ/stanza-plugins/pull/206))
  - Add default for `pod_name` parameter
- Update `nginx` plugin ([PR206](https://github.com/observIQ/stanza-plugins/pull/206))
  - Add default for `pod_name` parameter
- Update `tail` plugin ([PR206](https://github.com/observIQ/stanza-plugins/pull/206))
  - Add default for `multiline_line_start_pattern` parameter

### CODE REVIEW

File: plugins/tomcat.yaml [hunk 1]
```diff
@@ -19,7 +19,7 @@ parameters:
     label: Pod Name
     description: The pod name (without the unique identifier on the end)
     type: string
-    required: true
+    default: 'tomcat-*'
```
Comment (schmikei) [inline plugins/tomcat.yaml:22]: Shouldn't we also be setting the default using the go template?

File: plugins/nginx.yaml [hunk 2]
```diff
@@ -19,7 +19,7 @@ parameters:
     label: Pod Name
     description: The pod name (without the unique identifier on the end)
     type: string
-    required: true
+    default: 'nginx-*'
```
Comment (djaglowski) [inline plugins/nginx.yaml:22]: Does this plugin need to be versioned?

File: plugins/tomcat.yaml [hunk 3]
```diff
@@ -19,7 +19,7 @@ parameters:
     label: Pod Name
     description: The pod name (without the unique identifier on the end)
     type: string
```
Comment (djaglowski) [inline plugins/tomcat.yaml:21]: Does this plugin need to be versioned?

File: plugins/nginx.yaml (hunk 2 above)
Comment (ericwholt) [reply plugins/nginx.yaml:23]: It does thanks.

File: plugins/tomcat.yaml (hunk 3 above)
Comment (ericwholt) [reply plugins/tomcat.yaml:22]: It does thanks.

File: plugins/tomcat.yaml (hunk 1 above)
Comment (ericwholt) [reply plugins/tomcat.yaml:23]: I agree. Added defaults.

**Review by schmikei — APPROVED** [review]

**Pull request merged**